libkbfs/cr_chains.go

```diff
@@ -636,7 +637,13 @@ func newCRChains(ctx context.Context, cfg Config, rmds []*RootMetadata,
 			ccs.blockChangePointers[ptr] = true
 		}
 
-		for _, op := range rmd.data.Changes.Ops {
+		// Copy the rmd since CR will change the ops.
+		rmdCopy, err := rmd.deepCopy(cfg.Codec(), false)
```

Comment:
do you need to make a copy of the full RMD? Can you just use copy the privateData, or even just the ops array?